dask/dataframe/shuffle.py

```diff
@@ -495,6 +495,10 @@ def rearrange_by_column(
         if ignore_index:
             df2._meta = df2._meta.reset_index(drop=True)
         return df2
+    elif shuffle == "p2p":
+        from distributed.shuffle import rearrange_by_column_p2p
```

Comment:
https://github.com/dask/distributed/issues/5996#issuecomment-1078245372

Reply:
This has been in distributed since the skeleton was first merged.  We're safe here on import.